applications/crossbar/priv/couchdb/swagger/swagger.json

```diff
@@ -7829,6 +7835,15 @@
         "/v2/accounts/{ACCOUNT_ID}/ledgers/{ID}/debit": {
             "post": {}
         },
+        "/v2/accounts/{ACCOUNT_ID}/ledgers/{ID}/{ID}": {
```

Comment:
How is this different from the `"/v2/accounts/{ACCOUNT_ID}/ledgers/{LEDGER_ID}/{ID}"` below?

Also, are the debit/credit paths above correct?